tools/certificate_configuration/PEM-to-C-string.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import argparse
+from enum import Enum
+import sys
+
+class CredentialType(Enum):
+    PRIVATE_KEY = 1
+    CLIENT_CERT = 2
+    ROOT_CA_CERT = 3
+
+CREDS_MACRO_MAP = {
+    CredentialType.CLIENT_CERT: 'keyCLIENT_CERTIFICATE_PEM',
+    CredentialType.ROOT_CA_CERT: 'keyJITR_DEVICE_CERTIFICATE_AUTHORITY_PEM',
+    CredentialType.PRIVATE_KEY: 'keyCLIENT_PRIVATE_KEY_PEM'
+}
+
+# Prints the passed credential file as a C string. If a specific type of the
+# credential is passed, then its associated configuration macro
+# from demos/include/aws_clientcredential_keys.h is also mentioned.
+def printCredentialAsCString(file_path, type=None):
```

Comment:
Since the config is at a fixed location, is it also possible to just write the configs to that file similar to how CI does it? Although, one disadvantage I see with that approach is that the logic for putting those configs into that file might need changing whenever the config gets updated.

Reply:
I thought about it and it seemed like the right way would be to parse the header file for the macro before writing to the file. We can consider that as a future improvement

Reply:
I think we intentionally decided against it as auto-magically updating a header file increases the possibility of someone accidentally cheecking-in credentials.